gains_workshops.rst

```diff
@@ -0,0 +1,24 @@
+GAINS Workshops
+^^^^^^^^^^^^^^^^^^
```

Comment:
Not strictly necessary, but better to keep the title and the syntax-highlighting in sync
```suggestion
^^^^^^^^^^^^^^^
```